doc/nrf/shortcuts.txt

```diff
@@ -6,7 +6,7 @@
 .. |release_tt| replace:: ``v2.9.0-nRF54H20-rc1``
 .. |release_number_tt| replace:: ``2.9.0-nRF54H20-rc1``
 
-.. |jlink_ver| replace:: v7.94i
+.. |jlink_ver| replace:: v8.10f
 .. |jlink_ver_vsc| replace:: v7.94i
```

Comment:
What about this line ?

Still refers to v7.94i.

Reply:
Not yet to be updated.

Reply:
accepted